lib/qx/tool/cli/commands/Compile.js

```diff
@@ -326,6 +326,7 @@ qx.Class.define("qx.tool.cli.commands.Compile", {
       if (t.argv["app-name"])
         appNames = t.argv["app-name"].split(',');
       
+      let hasIndexHtmlToRoot = false;  
```

Comment:
The default should match the *current* behaviour. Which would be `true`.

Reply:
Default behaviour to generate.py or to new compiler? For new compiler default is false.

Reply:
I'd expect the new way to behave more or less like the old way. Mapping to a multi-application layout by default is a special usecase IMHO. As written in the issue itself, I'd not expect to be forced to some subdirectories.

Reply:
what's about ``` let hasIndexHtmlToRoot = (data.applications.length === 1);'''
Then we have root in a single application and will not break multi applications.

We will break all settings from people wich use new compiler.

Reply:
Whatever retains the 'old' beaviour in some way is welcome. There has not been any multi application setup before - at least not without tweaking around in the *config.json* file.

So if it's possible to keep people with a standard single application setup from facing a 'strange' file layout (at least for the most common usecase), would be really nice.

Alternatively the default `qx create application` command should create a configuration which is configured for single applications properly - but I still think that the default value for such a setting should be the most common one. Developers who want to create a multi application layout can configure it.